webpack.dev.js

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉